test-lib-openshift.sh

```diff
@@ -883,50 +665,31 @@ function ct_os_test_template_app_func() {
   # Upload main image is already done by CVP pipeline. No need to do it twice.
   if [ "${CVP:-0}" -eq "0" ]; then
     # Create a specific imagestream tag for the image so that oc cannot use anything else
-    if [ "${CT_OCP4_TEST:-false}" == 'true' ] ; then
```

Comment:
@zmiklank This variable should be present, The function `ct_run_tests_from_testset` is used for container tests as for openshift tests. And we need to switch to the default project in case of OpenShift tests. For container tests, it does not make sense.

Reply:
I see, thanks for explanation.